packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1582,6 +1600,9 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
     function _checkAccountLoad(
         address _address
     )
+        requiresNuisanceGas(
```

Comment:
This will never overlow correct? Perhaps a comment saying that would be nice here